config/jobs/kubernetes-sigs/kueue/kueue-presubmits-main.yaml

```diff
@@ -252,7 +252,7 @@ presubmits:
     cluster: eks-prow-build-cluster
     branches:
     - ^main
-    skip_if_only_changed: "^docs/|^\\.github/|\\.(md)$|^(README|LICENSE|OWNERS)$"
+    skip_if_only_changed: "^docs/|^\\.github/|^(README|LICENSE|OWNERS)$|^cmd/experimental/kjobctl/"
```

Comment:
`site` can still be excluded.

Reply:
No, in theory you could modify some generated files.

Reply:
gotcha, thanks